README.md

```diff
@@ -7,6 +7,173 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
+| Determined AI | Encouraged | Restricted | Restricted | Restricted | 2020-03-06 |
+| Digit | Required | Restricted | Restricted | Restricted | 2020-03-05 |
+| Dockwa | Required | Restricted | Restricted | Restricted | 2020-03-11 |
+| Dolby | Optional | Restricted | Restricted | Restricted | 2020-03-09 |
+| Donorschoose | Encouraged for NYC, Required for SF | Restricted | Restricted | Restricted | 2020-03-10 |
+| DrChrono | Required | Restricted | Restricted | Restricted | 2020-03-09 |
+| Drift | Required | Restricted | Restricted | Restricted | 2020-03-10 |
+| Drop | Required | Restricted | Restricted | Restricted | 2020-03-09 |
+| Dropbox | Encouraged | Restricted | Restricted | Restricted | 2020-03-06 |
+| Dstillery | Encouraged | Discouraged | Restricted | Restricted | 2020-03-10 |
+| eBay | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
+| Equinix | Encouraged for Seattle and Bay Area until 4/3 | Restricted| Restricted | Restricted | 2020-03-10 |
+| Expedia | Encouraged, some Required | Restricted | ? | ? | 2020-03-06 |
+| f5 Networks | Encouraged for Seattle and Bay Area until 3/23 | Restricted | Restricted | Restricted | 2020-03-05 |
+| Facebook[[1]](https://www.geekwire.com/2020/facebook-contractor-seattle-tests-positive-coronavirus-company-closes-office/) | Encouraged | Restricted | Restricted | Restricted | 2020-03-04 |
+| Fastly[[1]](https://www.fastly.com/blog/exercising-caution-as-covid-19-evolves) | Required | Restricted | Restricted | Restricted | 2020-03-01 |
+| Freshworks | Optional | Restricted | ? | ? | 2020-03-06 |
+| FullStory | Encouraged | Restricted | ? | Restricted | 2020-03-11 |
+| Fundbox | Required | Restricted | ? | ? | 2020-03-11 |
+| Gem | Optional | Restricted | Restricted | Restricted | 2020-03-11 |
+| General Dynamics | Optional | Restricted | ? | ? | 2020-03-11 |
+| GetYourGuide[[1]](https://twitter.com/JohannesReck/status/1237361981008592896) | Required | Restricted | Restricted | Restricted | 2020-03-11 |
+| GitHub[[1]](https://github.blog/2020-03-03-covid-19-update-supporting-our-employees-and-community/) | Encouraged | Restricted | ? | ? | 2020-03-04 |
+| Glassdoor[[1]](https://www.glassdoor.com/blog/glassdoor-coronavirus/) | Required | Restricted | Restricted | ? | 2020-03-05 |
+| Glitch | Required | Restricted | Restricted | Restricted | 2020-03-09 |
+| Google[[1]](https://www.theguardian.com/world/2020/mar/04/coronavirus-google-tech-dublin-twitter-work-from-home#maincontent) [[2]](https://www.reuters.com/article/us-health-coronavirus-alphabet/google-recommends-all-north-america-employees-to-work-from-home-idUSKBN20X38J?feedType=RSS&feedName=technologyNews) | Encouraged / Geo-specific | Restricted | Restricted | ? | 2020-03-11 |
+| Granular.ag | Required for SF / All others allowed | Restricted | Restricted for SF | Restricted | 2020-03-05 |
+| Groupon | Encouraged | Restricted | ? | Restricted | 2020-03-06 |
+| Guidewire | Encouraged | Restricted | Restricted | Restricted | 2020-03-11 |
```

Comment:
```suggestion
| Guidewire | Encouraged | Restricted | Restricted | Restricted | 2020-03-12 |
```

Reply:
Sorry, I meant *remove* all the companies that your PR is adding. GitHub auto-generates the list from the YAML, so if I merge this, we'll have duplicate companies.

In other words, your PR should have zero changes to README.md.

Reply:
Oh, I see. Done.